lib/std/os/linux/io_uring.zig

```diff
@@ -1260,6 +1260,36 @@ pub const IO_Uring = struct {
         }
     }
 
+    pub const IO_WQ_WorkerLimit = struct {
+        max_bounded_workers: u32,
+        max_unbounded_workers: u32,
+    };
+
+    // Sets an upper bound for workers used to perform work required for bounded work (e.g. regular file or block device
+    // operations) and unbounded work (i.e. requests that may never complete e.g. network requests)
+    // Passing 0 for one of the limits will leave the current limit unchanged.
+    // On success, returns the previously set limits
+    // Available since 5.15
+    // Note: These pools of workers are shared only among io_uring instances that have been created by the same thread,
+    //       in the same process and consequently the limits set with this function only affect io_uring instances at
+    //       this scope.
+    pub fn set_io_wq_worker_limits(self: *IO_Uring, new_limits: IO_WQ_WorkerLimit) !IO_WQ_WorkerLimit {
+        const data: [2]u32 = .{ new_limits.max_bounded_workers, new_limits.max_unbounded_workers };
+
+        const res = linux.io_uring_register(self.fd, .REGISTER_IOWQ_MAX_WORKERS, @as(*const anyopaque, @ptrCast(&data[0])), 2);
```

Comment:
Pretty sure you can just do `&data` or at worst `&data[0]`.
```suggestion
        const res = linux.io_uring_register(self.fd, .REGISTER_IOWQ_MAX_WORKERS, &data, 2);
```

Reply:
Giving the result type is definitely unnecessary. Didn´t know that @ptrCast is uneccessary though i.e. that any ptr coerces to a ptr to anyopaque, but I guess that makes sense.

Reply:
Right, the idea is that it's always safe to treat a particular pointer type as an opaque pointer. But the reverse operation is not safe, so that one requires a cast.